### ISSUE TITLE
Problem with HTML coloring

### ISSUE THREAD

**Issue by mniewrzal** [eclipse/tm4e#136]
I found HTML file where syntax coloring doesn't work to good. I attached screenshot and example file. Example works fine in VS Code and no stacktraces in logs.

![screenshot from 2017-09-05 10-10-56](https://user-images.githubusercontent.com/7313031/30051585-e25f6f6a-9222-11e7-8513-0e5f67e76ef6.png)

[test.txt](https://github.com/eclipse/tm4e/files/1276555/test.txt)

**Comment by angelozerr** [comment]
I suspect it's the same issue than https://github.com/eclipse/tm4e/issues/134#issuecomment-327043827 (encoding problem)

**Comment by angelozerr** [comment]
@mniewrzal could retry with last version of tm4e, thanks!

**Comment by angelozerr** [comment]
@mniewrzal  can we close this issue?

**Comment by mniewrzal** [comment]
Yes, look ok now :) Sorry for the delay, I was far from Eclipse things for some time.

:+1:
